projects/optic/src/commands/capture/actions/captureRequests.ts

```diff
@@ -180,11 +180,9 @@ function sendRequests(
     }
 
     return limiter.schedule(() =>
-      fetch(urljoin(proxyUrl, r.path), opts)
-        .then((response) => response.json())
```

Comment:
This is what was throwing the error, we were assuming everything was json and this threw an error when it wasn't json - the rest of the code sort of handled this